### PR TITLE
nGraph Bridge documentation updates 

### PR DESCRIPTION
**Pull request by indie**
- Update diagnostics `README` with link to latest debug and tracing tools on nGraph
- Fix various links that were not updated to point to tensorflow repo-managed content
- Refactor install methods into easy and standard options for simplified use cases

### CODE REVIEW

File: diagnostics/README.md [hunk 1]
```diff
@@ -1,65 +1,84 @@
...
+* You need to enable ngraph every time you called ```ngraph_bridge.disable()```, so it is good to check whether ngraph is enabled by calling ```ngraph.is_enabled()```
 * _Caution_: The above functions are only effective at the beginning of the execution. Once the session is created and ```run``` is called, the above functions will not be able to disable ngraph. 
 * For example usage, take a look at the ```model_test/verify_model.py``` in the diagnostics folder
 
 
 ## Protobuf Visualization
-The python script ngtf_graph_viewer.py can convert a protobuf (pb or pbtxt) into a dot file or a TB log, which can be viewed using TB. If the input is a pbtxt then ngtf_graph_viewer can also sanitize node names to remove underscores from the front of node names (which indicate they are internal nodes and might cause TB to complain). It can also prepend strings in front of certain node names, a feature which can be used  to append encapsulate information for clustering nodes together
 
-ngtf_graph_viewer has been tested on Python 2 TF-1.9, but should work with Python 3 and other versions of TF.
+The [ngtf_graph_viewer] script can convert a protobuf (`pb` or `pbtxt`) into a
```
Comment (avijit-nervana) [inline diagnostics/README.md]: We will remove this script most likely.

File: README.md [hunk 2]
```diff
@@ -3,39 +3,46 @@
...
 
-# Intel(R) nGraph(TM) Compiler and Runtime for TensorFlow*
-
-This repository contains the code needed to enable Intel(R) nGraph(TM) Compiler and 
-runtime engine for TensorFlow. Use it to speed up your TensorFlow training and 
-inference workloads. The nGraph Library and runtime suite can also be used to 
-customize and deploy Deep Learning inference models that will "just work" with 
-a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the 
-[Intel(R) Nervana(TM) NNP](https://itpeernetwork.intel.com/inteldcisummit-artificial-intelligence/).
+#  Bridge to nGraph Compiler stack
```
Comment (avijit-nervana) [inline README.md]: Keep the contents of this readme as is. Only fix the formatting (mentioned below)

File: README.md [hunk 3]
```diff
@@ -98,33 +129,33 @@ When the build finishes, a new `virtualenv` directory is created in `build_cmake
 
 Add the following flags to build PlaidML and Intel GPU backends (optional):
 
-        --build_plaidml_backend
-        --build_intelgpu_backend
+    --build_plaidml_backend
```
Comment (avijit-nervana) [inline README.md]: Just add these fixes.

File: diagnostics/README.md [hunk 4]
```diff
@@ -1,65 +1,84 @@
...
 ## Debug flags
-* ```NGRAPH_ENABLE_SERIALIZE=1```: Generate nGraph level serialized graphs .json
-* ```NGRAPH_CPU_TRACING=1```: Generate nGraph level function timelines
-* ```NGRAPH_TF_VLOG_LEVEL=5```: Generate ngraph-tf logging info for different passes
-* ```NGRAPH_TF_LOG_PLACEMENT=1```: Generates op placement log at stdout
-* ```NGRAPH_TF_DUMP_CLUSTERS=1```: Dumps Encapsulated TF Graphs: ngraph_cluster_<cluster_num>
-* ```NGRAPH_TF_DUMP_GRAPHS=1```: Dumps TF graphs for different passes : precapture, capture, unmarked, marked, clustered, declustered, encapsulated
-* ```TF_CPP_MIN_VLOG_LEVEL=1```: Enables TF CPP Logs 
-* ```NGRAPH_TF_DUMP_DECLUSTERED_GRAPHS=1```: Dumps graphs with final clusters assigned. Use this to view TF computation graph with colored nodes indicating clusters
+
```
[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: Was going for consistency with all of the other examples in the directory:  
diagnostics_readme

File: diagnostics/README.md [hunk 5]
```diff
@@ -1,65 +1,84 @@
...
+* protobuf binary: `pb`
+* protobuf string text: `pbtxt`
+* NGraph-TF: **NGTF**
 
-## What to do if your network fails
-**TODO**: List steps to generate logs or run diagnostic tools
+
+## What to do if your network fails?
+
+See the [nGraph Library documentation] for how to enable tracing and logs.
```
[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: It already does have this; are you saying we should not link to it? http://ngraph.nervanasys.com/docs/latest/frameworks/generic-configs.html#activate-logtrace-related-environment-variables

File: diagnostics/README.md (hunk 1 above)
Comment (indie) [reply diagnostics/README.md]: Done

File: diagnostics/README.md [hunk 6]
```diff
@@ -1,65 +1,84 @@
 # Diagnostics tools
 
-A collection of tools to help triage and diagnose misbehaving networks.
+What follows here is a collection of tools to help triage and diagnose 
+misbehaving networks.
```
[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: Done

File: diagnostics/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: Done, we now have a table.

[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: Also realized I forgot to sort alphabetically.  :abcd:

File: diagnostics/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (avijit-nervana) [reply diagnostics/README.md]: Revieing the link, I am not sure that the nGraph documentation will help unless the user can debug at the bridge level. So I will leave it unchanged.

[the remark this answers is not in the document]
Comment (indie) [reply diagnostics/README.md]: Thank you, @avijit-nervana.  File updated.

**Pull request merged**